hawkular-alerts-rest-tests/pom.xml

```diff
@@ -219,7 +219,6 @@
               <excludes>
                 <excludeFile>org/hawkular/alerts/rest/ClusterITest.class</excludeFile>
                 <excludeFile>org/hawkular/alerts/rest/PerfCrudITest.class</excludeFile>
-                <exclude>${hawkular.itest-exclude}</exclude>

```

Comment:
I think it was showing an error in the editor and I don't remember if it was used in this profile.
I can bring it back if it was a mistake.